e2e/testdata/fn-render/resource-has-pkgname-prefix/Kptfile

```diff
@@ -4,6 +4,6 @@ metadata:
   name: wordpress
 pipeline:
   mutators:
-  - image: gcr.io/kpt-fn/set-annotation:unstable
+  - image: gcr.io/sunilarora-sandbox/set-annotation:0.1
```

Comment:
I will update to the one from `gcr.io/kpt-fn/...` once it is there, using it for testing for now.

Reply:
Done.